extension-config.json

```diff
@@ -26,7 +26,9 @@
       "default": {
         "enabled": true,
         "ratio": 0.4,
-        "styles": null
+        "styles": {
+          "filter": "blur(3px)"
```

Comment:
You need filter only images, not other nodes. The filtering and transitioning at the same time lead to concurrency of rendering and produces blinking (as on video)